Cargo.toml

```
@@ -13,45 +13,94 @@ bench = false
+maud = { version = "0.25.0", default-features = false, features = [
+    "actix-web",
+] }
+scraper = { version = "0.18.1", default-features = false }
+actix-web = { version = "4.4.0", features = [
+    "cookies",
+    "macros",
+    "compress-brotli",
+], default-features = false }
+actix-files = { version = "0.6.5", default-features = false }
+actix-cors = { version = "0.7.0", default-features = false }
+fake-useragent = { version = "0.1.3", default-features = false }
+env_logger = { version = "0.11.1", default-features = false }
+log = { version = "0.4.21", default-features = false }
+mlua = { version = "0.9.1", features = [
+    "luajit",
+    "vendored",
+], default-features = false }
+redis = { version = "0.24.0", features = [
+    "tokio-comp",
+    "connection-manager",
+], default-features = false, optional = true }
+blake3 = { version = "1.5.0", default-features = false }
+error-stack = { version = "0.4.0", default-features = false, features = [
+    "std",
+] }
+async-trait = { version = "0.1.76", default-features = false }
+regex = { version = "1.9.4", features = ["perf"], default-features = false }
+smallvec = { version = "1.13.1", features = [
+    "union",
+    "serde",
+], default-features = false }
+futures = { version = "0.3.30", default-features = false, features = ["alloc"] }
+dhat = { version = "0.3.2", optional = true, default-features = false }
 mimalloc = { version = "0.1.38", default-features = false }
-async-once-cell = {version="0.5.3", default-features=false}
-actix-governor = {version="0.5.0", default-features=false}
-mini-moka = { version="0.10", optional = true, default-features=false, features=["sync"]}
-async-compression = { version = "0.4.6", default-features = false, features=["brotli","tokio"], optional=true}
-chacha20poly1305={version="0.10.1", default-features=false, features=["alloc","getrandom"], optional=true}
-chacha20 = {version="0.9.1", default-features=false, optional=true}
-base64 = {version="0.21.5", default-features=false, features=["std"], optional=true}
-cfg-if = {version="1.0.0", default-features=false,optional=true}
+async-once-cell = { version = "0.5.3", default-features = false }
+actix-governor = { version = "0.5.0", default-features = false }
+mini-moka = { version = "0.10", optional = true, default-features = false, features = [
+    "sync",
+] }
+async-compression = { version = "0.4.6", default-features = false, features = [
+    "brotli",
+    "tokio",
+], optional = true }
+chacha20poly1305 = { version = "0.10.1", default-features = false, features = [
+    "alloc",
+    "getrandom",
+], optional = true }
+chacha20 = { version = "0.9.1", default-features = false, optional = true }
+base64 = { version = "0.21.5", default-features = false, features = [
+    "std",
+], optional = true }
+cfg-if = { version = "1.0.0", default-features = false, optional = true }
+keyword_extraction = { version = "1.3.0", default-features = false, features = [
+    "tf_idf",
+
+
+] }
+
+stop-words = "0.8.0"
```

```suggestion
stop-words = { version = "0.8.0", default-features = false, features = [
     "iso",
]}
```
I would suggest disabling the default features for the `stop-words` crate as this will not enable any non-essential features when building/compiling the app, which can help reduce the build time significantly. :slightly_smiling_face: